agent/docker-compose.yml

```diff
@@ -37,15 +37,13 @@ services:
       - "RABBIT_CAPABILITIES=bert,no_sort"
 
   citamon_agent_cita_exporter:
-    build: ./cita_exporter
-    image: citamon/agent-cita-exporter
+    image: citamon/agent-cita-exporter:20.2.2
```

Comment:
why using tag `20.2.2`? the latest release version of CITA is `v20.2.0`

Reply:
@0xA0000

Reply:
It's the version number of cita-cli, and I think it should be. The tag represents which version of a tool that we use to collect metrics in containers. Like we did in rabbitmq-exporter and process-exporter and host-exporter.

Reply:
we need to get repo "citamon/agent-cita-exporter" source opened, put it into a sub dir